controllers/articles.controllers.js

```diff
@@ -0,0 +1,18 @@
+const { fetchArticle } = require("../models/articles.models");
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+
+  let regex = /\d/;
+  if (!regex.test(article_id)) {
+    res.status(400).send({ message: "invalid request" });
+  }
```

Comment:
you can handle case where article_id is not a number in the PSQL error handler as psql will give back an error